src/components/Dashboard/Content/index.tsx

```diff
@@ -47,22 +48,26 @@ const DashboardContent = ({
       </div>
     </div>
   ) : (
-    <div className="flex flex-col h-full" data-test-id={dashboardDataTestIds.content}>
+    <div className="flex flex-col h-full" data-test-id={dashboardContentDataTestIds.base}>
       <div
-        style={{ backgroundImage: `url(${getHighResolutionImage(surveys[currentPosition].coverImageUrl)})` }}
+        style={{
+          backgroundImage: `url(${surveys.length ? getHighResolutionImage(surveys[currentPosition].coverImageUrl) : ''})`,
+        }}
         className="w-full h-[302px] rounded-[12px] bg-cover duration-500 ease-in-out"
       ></div>
       <div className="flex flex-row justify-between mt-[38px]">
         <div className="flex flex-col justify-between">
           <p className="text-white text-x-regular font-extrabold">{surveys[currentPosition].title}</p>
           <p className="text-white text-regular tracking-survey-tight opacity-60 mt-2">{surveys[currentPosition].description}</p>
         </div>
-        <button
-          type="button"
-          className="w-[56px] h-[56px] bg-white rounded-full inline-flex items-center justify-center text-black-chinese"
-        >
-          <ArrowRight />
-        </button>
+        <Link to={`surveys/${surveys[currentPosition].id}`}>
```

Comment:
be careful, in the line 54, I saw there are some potential situations that `surveys` might be empty. It could break the app when you extract data like this.
There are 2 suggestions
1. Add a high-level check for `surveys` existence
2. Using `lodash/get` function,
```suggestion
        <Link to={`surveys/${_get(surveys, 'currentPosition.id', '')}`}>
```

Reply:
Good catch, I will go with option 2 at this moment, thanks for the tip 🤑

Reply:
@nvminhtue updated in [90da065](https://github.com/manh-t/react-survey/pull/61/commits/90da0659186ddd63cbab8439831f1b090a9eee7c)